packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -85,17 +85,28 @@ const fetchLinkSuggestions = async (
 ) => {
 	const {
 		isInitialSuggestions = false,
+		initialSuggestionsOptions = undefined,
```

Comment:
```suggestion
		initialSuggestionsSearchOptions = undefined,
```

Nit: shall we be ultra clear in our naming?